### PR TITLE
[tests-only]Bump core latest commit-id on reva-master

### PR DESCRIPTION
**Pull request by SagarGi**
### Description

Bump `Core` latest commit-id to reva-master

### Related Issue:
https://github.com/owncloud/QA/issues/772

### CODE REVIEW

File: tools/prepare-release/main.go [hunk 1]
```diff
@@ -71,10 +71,6 @@ func main() {
 	cmd = exec.Command("mv", "changelog/unreleased", newChangelog)
 	run(cmd)
 
-	// install release-deps: calens
-	cmd = exec.Command("make", "release-deps")
-	run(cmd)
-
```
Comment (phil-davis) [inline tools/prepare-release/main.go:77]: @SagarGi "something happened" - lines like this should not be deleted in a tests-only PR.

File: tools/prepare-release/main.go [hunk 2]
```diff
@@ -196,12 +192,6 @@ func createCommit(msg string) {
 }
 
 func createTag(version string) {
-	// check if repo is dirty
-	if isRepoDirty() {
-		fmt.Fprintf(os.Stderr, "repo is dirty when creating a new tag for version %s", version)
-		os.Exit(1)
-	}
-
```
Comment (phil-davis) [inline tools/prepare-release/main.go:204]: @SagarGi "something happened" - lines like this should not be deleted in a tests-only PR.

**Review by phil-davis — COMMENTED** [review]
There are other lines deleted.

**Comment by SagarGi** [comment]
@phil-davis  https://github.com/cs3org/reva/pull/3476/commits/1f9d7f198f581174b4f78397e4d1b5d0cfa6597c , https://github.com/cs3org/reva/pull/3476/commits/af911b7f59eae36d4f3262e9a4ad037dee1701be commits were added by @gmgigi96  shall i remove those ? or what should i do ?

**Comment by phil-davis** [comment]
> @phil-davis [1f9d7f1](https://github.com/cs3org/reva/commit/1f9d7f198f581174b4f78397e4d1b5d0cfa6597c) , [af911b7](https://github.com/cs3org/reva/commit/af911b7f59eae36d4f3262e9a4ad037dee1701be) commits were added by @gmgigi96 shall i remove those ? or what should i do ?

Yes, remove them. They somehow got into this PR accidentally.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**